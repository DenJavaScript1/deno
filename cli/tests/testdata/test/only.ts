Deno.test({
  name: "before",
  fn() {},
});

Deno.test({
  only: true,
  name: "only",
  fn() {},
});

Deno.test("only", () => {}, {
  only: true,
});

Deno.test({
  name: "after",
  fn() {},
});
