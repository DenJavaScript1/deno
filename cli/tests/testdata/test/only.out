Check [WILDCARD]/test/only.ts
running 2 tests from [WILDCARD]/test/only.ts
test only ... ok ([WILDCARD])
test only ... ok ([WILDCARD])

test result: ok. 2 passed; 0 failed; 0 ignored; 0 measured; 2 filtered out ([WILDCARD])

error: Test failed because the "only" option was used
